src/aiidalab_qe/workflows/__init__.py

```diff
@@ -121,8 +122,24 @@ def get_builder_from_protocol(
             parameters["advanced"]["pw"]["pseudos"][kind] = orm.load_node(uuid)
         #
         builder = cls.get_builder()
-        # Set the structure.
-        builder.structure = structure
+        # Set a HubbardStructureData if hubbard_parameters is specified
+        hubbard_dict = parameters["advanced"].pop("hubbard_parameters", None)
+        if hubbard_dict is not None:
+            hubbard_parameters = hubbard_dict["hubbard_u"]
+            hubbard_structure = HubbardStructureData.from_structure(structure)
+            for key, value in hubbard_parameters.items():
+                kind, orbital = key.rsplit("-", 1)
```

Comment:
There are spaces when creating the label.
```suggestion
                kind, orbital = key.rsplit(" - ", 1)
```

Reply:
Done